Bottom: a019c8d4e13afeb68aa3a22b095b9d127ba4e7d6
Top:    d9b1a287fc58d02a8717a42f9067cb05bc98da10
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-21 17:05:03 +0800

Refresh of tuntap-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index d8f2912..43811e7 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1626,7 +1626,7 @@ static int tun_sendmsg(struct socket *sock, struct msghdr *m, size_t total_len)
 		return -EBADFD;
 
 	ret = tun_get_user(tun, tfile, m->msg_control, &m->msg_iter,
-			   m->msg_flags & MSG_DONTWAIT)
+			   m->msg_flags & MSG_DONTWAIT);
 	tun_put(tun);
 	return ret;
 }
